eng/Baseline.Designer.props

```diff
@@ -603,7 +603,7 @@
     <BaselinePackageVersion>3.1.0</BaselinePackageVersion>
   </PropertyGroup>
   <ItemGroup Condition=" '$(PackageId)' == 'Microsoft.AspNetCore.SignalR.StackExchangeRedis' AND '$(TargetFramework)' == 'netcoreapp3.1' ">
-    <BaselinePackageReference Include="MessagePack" Version="[1.7.3.7, )" />
+    <BaselinePackageReference Include="MessagePack" Version="[2.0.323, )" />
```

Comment:
Revert

Reply:
-_- i though i reverted it, sorry about the noise here